MeesGame/Gameplay/GameObjects/TilesTypes/EndTile.cs

```diff
@@ -10,5 +10,21 @@ public override void EnterCenterOfTile(ITileFieldPlayer player)
         {
             player.Win();
         }
+
+        public override bool CanPlayerMoveHere(ITileFieldPlayer player)
+        {
+            return true;
+        }
+
+        public override void UpdateGraphicsToMatchSurroundings()
+        {
+            int x = Location.X;
+            TileField tileField = Parent as TileField;
+
+            if (x == 0 || x == tileField.Columns)
+            {
+                sprite = new SpriteSheet("verticalEnd");
+            }
```

Comment:
Add an else to make it a horizontal exit. Don't rely on earlier states of the tile.